sim/core/unit.go

```diff
@@ -160,6 +160,27 @@ func (character *Character) DoNothing() {
 	character.doNothing = true
 }
 
+func (unit *Unit) IsActive() bool {
```

Comment:
`IsEnabled` has already been moved into `Unit`.

Reply:
Did not see that will use it instead 👍